library.js

```diff
@@ -0,0 +1,123 @@
+const myLibrary = [];
+
+
+class Book {
+  constructor(title, author, pages, read) {
+    this.title = title;
+
+    this.author = author;
+
+    this.pages = pages;
+
+    this.read = read;
+  }
+
+  isRead(render) {
+    this.read = !this.read;
+    render();
+  }
+
+  static deleteBook(index, render) {
+    myLibrary.splice(index, 1);
+    render();
+  }
+}
+
+const clearElement = (element) => {
+  element.innerHTML = '';
+};
+
+function addBookToLibrary() {
+  const title = document.querySelector('#title').value;
+
+  const author = document.querySelector('#author').value;
+
+  const pages = document.querySelector('#pages').value;
+
+  const read = document.querySelector('#read').checked;
+
+  const newbook = new Book(title, author, pages, read);
+
+  myLibrary.push(newbook);
+
+  clearElement(document.querySelector('#form'));
+  // eslint-disable-next-line no-use-before-define
+  render();
+}
+
+const renderForm = () => {
+  const form = document.querySelector('#form');
+
+  form.innerHTML = `<label for="title">Title</label> 
+
+  <input type="text" name="title" id="title" />
+  <br>
+  <label for="author">Author</label>
+
+  <input type="text" name="author" id="author" />
+  <br>
+  <label for="pages">Pages</label>
+
+  <input type="number" name="pages" id="pages" />
+  <br>
+  <label for="read">Read</label>
+
+  <input type="checkbox" name="read" id="read" />
+  <br>
+  <button type="submit" id= 'create'>Create Book</button>`;
```

Comment:
You need to do form controls to make sure empty forms cannot be submitted
![image](https://user-images.githubusercontent.com/31897434/87428234-f2c82700-c5d9-11ea-883d-7920b82f0343.png)